examples/readme.md

```diff
@@ -23,3 +23,9 @@ This section contains examples using the pretrained checkpoints and pre-calculat
 This section contains examples using attribution scores to estimate the brittleness of a model.
 
 [Use influence function to estimate the brittleness of losigitc regression trained on Mnist10](./brittleness/mnist_lr_brittleness.py)
+
+## Data selection
```

Comment:
I guess we can call this application "data cleaning"?
which is more consistent to what we understand (data selection if more like the opposite of this) and also our survey paper's tone

Reply:
good idea. I can change it to data cleaning for better understanding of our users.

Reply:
solved